guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,319 @@
+---
+layout: guide
+title: Lightning Services
+description: Third party services that solve common user experience problems when connecting to and using the lightning network.
+nav_order: 4
+parent: How it works
+permalink: /guide/how-it-works/lightning-services/
+redirect_from:
+ - /guide/payments/send/lightning-service-providers/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/lightning-services.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-services/lightning-services@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/lightning-services-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/lightning-services-mobile@2x.jpg"
+   alt-text = "Lightning service providers header image."
+   caption = "Image by [Rahul Viswanath](https://unsplash.com/@rahul_viswanath) from Unsplash"
+   width = 1600
+   height = 740
+   layout = "full-width"
+%}
+
+# Lightning services
+
+[Scaling]({{ '/guide/getting-started/technology-primer/#do-all-transactions-have-to-be-this-secure' | relative_url }}) bitcoin to billions of people non-custodially with the lightning network creates many new usability issues. Including liquidity and uptime requirements, channel management, routing, and more complex backups. Lightning services offered by third-parties aim to solve these issues.
+
+In this section, we cover two categories of lightning services. Lightning service providers (LSPs), which focus on onboarding and providing connectivity to lightning users, and [Lightning wallet servers]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers' | relative_url }}), which provide additional services not relating to connectivity.
+
+## What is a Lightning service provider?
```

Comment:
lowercase consistency
```suggestion
## What is a lightning service provider?
```

Reply:
Not sure about this one as an LSP, like an ISP, is a proper name

Reply:
Not a blocker for me, as you say.

Reply:
Lighting service provider and lightning (which I lowercase) are two separate things